### PR TITLE
[BE-01] Add cooking_time field to recipes model, add validation & response

### PR DESCRIPTION
**Pull request by dansob14**
[Add cooking_time field to recipes model, add validation & response](https://trello.com/c/xQ8Ljkxm/1-be-01-add-cookingtime-to-recipes-return-it-in-response-l)
Add cooking_time field to recipes model, add validation & response.

AC:
1. Cooking_time field may be set during create or update a record
2. Cooking_time field is returned in #show response.
3. Cooking_time field has a validation for checking unrealistic cases
4. Cooking field returns ‘unknow’ if value is empty.

### CODE REVIEW

File: app/serializers/recipe_serializer.rb [hunk 1]
```diff
@@ -1,3 +1,4 @@
 class RecipeSerializer < ActiveModel::Serializer
   attributes :id, :name, :content
+  UNKNOWN = 'unknown'
 end
```
Comment (grabomarcin) [inline app/serializers/recipe_serializer.rb]:
```suggestion
class RecipeSerializer < ActiveModel::Serializer
  UNKNOWN = 'unknown'
  attributes :id, :name, :content
end
```

File: skrypt.sh [hunk 2]
```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+
+IP_ADDR=`/sbin/ifconfig eth0 | grep 'inet addr' | cut -d: -f2 | awk '{print $1}'`
+
+echo "Twoj publiczny adres ip to: "
+curl http://169.254.169.254/latest/meta-data/public-ipv4 
+echo ""
+rails s -b $IP_ADDR -p 8082
```
Comment (grabomarcin) [inline skrypt.sh]:
```suggestion
rails s -b $IP_ADDR -p 8082

```

File: spec/controllers/api/v1/recipes_controller_spec.rb [hunk 3]
```diff
@@ -45,7 +47,15 @@
     context 'with invalid params' do
       it 'doesn\'t create recipe' do
         expect do
-          post :create, params: { recipe: { name: '', content: '' } }
+          post :create, params: { recipe: { name: '', content: '', cooking_time: 'invalid' } }
```
Comment (grabomarcin) [inline spec/controllers/api/v1/recipes_controller_spec.rb]: You can extract params to let:()

File: spec/controllers/api/v1/recipes_controller_spec.rb [hunk 4]
```diff
@@ -69,6 +101,19 @@
...
 
+    context 'with valid cooking_time' do
+      it 'updates cooking_time' do
+        expect do
+          put :update, params: { id: recipe.id, recipe: { cooking_time: new_cooking_time } }
+        end.to change { recipe.reload.cooking_time }.from(old_cooking_time).to(new_cooking_time)
+      end
+
+      it 'returns updated object' do
+        put :update, params: { id: recipe.id, recipe: { cooking_time: new_cooking_time } }
```
Comment (plokijro) [inline spec/controllers/api/v1/recipes_controller_spec.rb]: move it to `let`

File: spec/controllers/api/v1/recipes_controller_spec.rb [hunk 5]
```diff
@@ -78,6 +123,16 @@
...
+
+    context 'with invalid cooking_time' do
+      let(:invalid_cooking_time) { nil }
+
+      it 'doesn\'t update name' do
+        expect do
+          put :update, params: { id: recipe.id, recipe: { cooking_time: invalid_cooking_time } }
+        end.not_to change { recipe.reload.cooking_time }
+      end
+    end
```
Comment (plokijro) [inline spec/controllers/api/v1/recipes_controller_spec.rb]: `cooking_time` can't be empty?

**Review by grabomarcin — APPROVED** [review]

**Pull request merged**